java/test/jmri/jmrix/jmriclient/JMRIClientTurnoutTest.java

```diff
@@ -43,26 +45,28 @@ public void testDispose() {
 
     @Test
     @Override
-    @Ignore("requires work for jmriclient turnouts")
-    public void testDirectFeedback() throws jmri.JmriException {
+    public void testDirectFeedback() throws Exception {
+
+	t.setFeedbackMode(Turnout.DIRECT);
```

Comment:
Isn't this bypassing the test of "Default feedback mode is Direct"?  

One could argue that I.e. jmriclient turnouts are not obeying that part of the definition of a turnout.  Except I don't see "Default feedback mode is Direct" documented anywhere in the Javadocs, so it might be more of a convention than a specification.

Reply:
Right, I don't see that documented anywhere either.   That was an assumption made (by me) when the base class test was written, which apparently was wrong.

It looks like our systems that implement monitoring (since not all do) default to Monitoring mode.

Reply:
Thanks! I hadn't remembered about monitoring.